packages/react-storage/package.json

```diff
@@ -71,7 +71,7 @@
       "name": "createStorageBrowser",
       "path": "dist/esm/browser.mjs",
       "import": "{ createStorageBrowser }",
-      "limit": "21 kB",
+      "limit": "30 kB",
```

Comment:
Should this be sufficient for remaining work? Or is this just so it can be ignored for a while?

Reply:
Either way, it makes sense to me to include a big bump and then reassess this once development is complete / if we run into the limit again